Hackney.Shared.Asset.Tests/Factories/EntityFactoryTest.cs

```diff
@@ -107,5 +107,161 @@ public void CanMapAnAssetTenureToAnAssetTenureDb(bool nullEndDate)
 
             entity.Should().BeEquivalentTo(databaseEntity);
         }
+
+        #region Asset Characteristics
+        [Fact]
+        public void AssetCharacteristicsMapsFromDatabaseToDomain()
+        {
+            // arrange
+            var databaseAC = _fixture.Create<AssetCharacteristicsDb>();
+
+            // act
+            var domainAC = databaseAC.ToDomain();
+
+            // assert
+            domainAC.NumberOfBedrooms.Should().Be(databaseAC.NumberOfBedrooms);
```

Comment:
Would it not be easier for something like:
`domainAC.Should().BeEquivalentTo(databaseAC)`

Reply:
Domain and Database layers have different concerns & while currently the models of the 2 layers are the same, in general they should not be equivalent & are expected to diverge by the architecture. As such, "BeEquivalent" would be a wrong assertion to do.